packages/webpack-loader/src/extract-plugin.ts

```diff
@@ -44,6 +44,7 @@ const forceCSSIntoOneStyleSheet = (compiler: Compiler) => {
       // We merge only CSS from Compiled.
       test: /css-loader\/compiled-css\.css$/,
       enforce: true,
+      priority: Infinity,
```

Comment:
This is not related to the SSR side effects, but it is an additional patch we had to do to make stylesheet extraction work